File: Largest_of_three_no/Largest_of_three_no.py
def Largest_of_three_no(a,b,c):
    if(a>b)and(a>c):
        return a
    elif(b>a)and(b>c):
        return b
    elif(c>a)and(c>b):
        return c
    else:
        return ("numbers are equal")

print(Largest_of_three_no(1,2,3))

File: Largest_of_three_no/Largest_of_three_no.cpp
#include <iostream>
using namespace std;

void Largest_of_three_no(int a,int b,int c){
    if (a > b && a > c){
        cout << "largest number is : " << a << endl;
    }
    else if (b > a && b > c){
        cout << "largest number is : " << b << endl;
    }
    else if (c > a && c > b){
        cout << "largest number is : " << c << endl;
    }
    else{
        cout << "numbers are equal or invalid \n";
    }
}

int main(int argc, char const *argv[])
{
    Largest_of_three_no(1,2,3);   
    return 0;
}
